tfx/components/evaluator/executor_test.py

```diff
@@ -28,6 +28,7 @@
 from tfx.proto import evaluator_pb2
 from tfx.types import artifact_utils
 from tfx.types import standard_artifacts
+from tfx.utils import json_utils
 
 
```

Comment:
Do we have tests with multiple splits?

Reply:
i updated the first test to include both 'train' and 'eval' for example_splits. is this what you were looking for?